Instructions for installing a specific CLI version (#3163)
dapr/docs: daprdocs/content/en/getting-started/install-dapr-cli.md

wget -q https://raw.githubusercontent.com/dapr/cli/master/install/install.sh -O - | /bin/bash
```

##### Installing a specific CLI version

> This example shows how to install CLI version `1.9.1`. Release candidates can be installed similarly, simply specify the version, e.g. `1.10.0-rc.3`.

Reviewer: ```suggestion
The following example shows how to install CLI version `1.9.1`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).
```

> This example shows how to install CLI version `1.9.1`. Release candidates can be installed similarly, simplify specify the version, e.g. `1.10.0-rc.3`

Author: ```suggestion
This example shows how to install CLI version `1.9.1`. Release candidates can be installed similarly, simply specify the version, e.g. `1.10.0-rc.3`
```

> The following example shows how to install CLI version `1.9.1`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).

Reviewer: ```suggestion
The following example shows how to install CLI version `{{% dapr-latest-version cli="true" %}}`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).
```

wget -q https://raw.githubusercontent.com/dapr/cli/master/install/install.sh -O - | DAPR_INSTALL_DIR="$HOME/dapr" /bin/bash
```

##### Installing a specific CLI version without `sudo`

> This example shows how to install CLI version `1.9.1`. Release candidates can be installed similarly, simply specify the version, e.g. `1.10.0-rc.3`.

Reviewer: ```suggestion
The following example shows how to install CLI version `1.9.1`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).
```

> This example shows how to install CLI version `1.9.1`. Release candidates can be installed similarly, simplify specify the version, e.g. `1.10.0-rc.3`

Author: ```suggestion
This example shows how to install CLI version `1.9.1`. Release candidates can be installed similarly, simply specify the version, e.g. `1.10.0-rc.3`
```

> The following example shows how to install CLI version `1.9.1`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).

Reviewer: ```suggestion
The following example shows how to install CLI version `{{% dapr-latest-version cli="true" %}}`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).
```

**Note:** Updates to PATH might not be visible until you restart your terminal application.

##### Installing a specific CLI version

> This example shows how to install CLI version `1.9.1`. Release candidates can be installed similarly, simply specify the version, e.g. `1.10.0-rc.3`.

Reviewer: ```suggestion
The following example shows how to install CLI version `1.9.1`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).
```

> The following example shows how to install CLI version `1.9.1`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).

Reviewer: ```suggestion
The following example shows how to install CLI version `{{% dapr-latest-version cli="true" %}}`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).
```

$script=iwr -useb https://raw.githubusercontent.com/dapr/cli/master/install/install.ps1; $block=[ScriptBlock]::Create($script); invoke-command -ScriptBlock $block -ArgumentList "", "$Env:DAPR_INSTALL_DIR"
```

#### Installing a specific CLI version without administrative rights

> This example shows how to install CLI version `1.9.1`. Release candidates can be installed similarly, simply specify the version, e.g. `1.10.0-rc.3`.

Reviewer: ```suggestion
The following example shows how to install CLI version `1.9.1`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).
```

> The following example shows how to install CLI version `1.9.1`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).

Reviewer: ```suggestion
The following example shows how to install CLI version `{{% dapr-latest-version cli="true" %}}`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).
```

curl -fsSL https://raw.githubusercontent.com/dapr/cli/master/install/install.sh | /bin/bash
```

##### Installing a specific CLI version

> This example shows how to install CLI version `1.9.1`. Release candidates can be installed similarly, simply specify the version, e.g. `1.10.0-rc.3`.

Reviewer: ```suggestion
The following example shows how to install CLI version `1.9.1`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).
```

> The following example shows how to install CLI version `1.9.1`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).

Reviewer: ```suggestion
The following example shows how to install CLI version `{{% dapr-latest-version cli="true" %}}`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).
```

curl -fsSL https://raw.githubusercontent.com/dapr/cli/master/install/install.sh | DAPR_INSTALL_DIR="$HOME/dapr" /bin/bash
```

##### Installing a specific CLI version without `sudo`

> This example shows how to install CLI version `1.9.1`. Release candidates can be installed similarly, simply specify the version, e.g. `1.10.0-rc.3`.

Reviewer: ```suggestion
The following example shows how to install CLI version `1.9.1`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).
```

> The following example shows how to install CLI version `1.9.1`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).

Reviewer: ```suggestion
The following example shows how to install CLI version `{{% dapr-latest-version cli="true" %}}`. You can also install release candidates by specifying the version (for example, `1.10.0-rc.3`).
```
